src/DataValues/MatrixValue.php

```diff
@@ -0,0 +1,128 @@
+<?php
+namespace DataValues;
+
+use DataValues\DecimalValue;
+/**
+ * Class representing a matrix.
```

Comment:
Please be a little more specific of what that means, e.g. "implemented as an array of arrays of numbers".